cmd/experimental/kueue-viz/backend/go.mod

```diff
@@ -0,0 +1,73 @@
+module github.com/akram/kueue-viz-go
```

Comment:
can we change to sigs.k8s.io/kueue/cmd/experimental/kueue-viz ? or just `kueue-viz`?

Reply:
done